components/config-mgmt-service/backend/elastic/elastic.go

```diff
@@ -31,7 +31,6 @@ const (
 	// Elasticsearch fields we use within this package
 	RunFieldTimestamp    = "start_time"
 	RunFieldEndTimestamp = "end_time"
-	CheckinTimestamp     = "checkin"
```

Comment:
The various codepaths used by the export endpoint were using "CheckinTimestamp and `backend.CheckIn` interchangeably, and it was confusing to me as someone unfamiliar with the code that this value could come from two places.